db/migrate/20170115193153_add_bought_to_order_items.rb

```diff
@@ -0,0 +1,5 @@
+class AddBoughtToOrderItems < ActiveRecord::Migration[5.0]
+  def change
+    add_column :order_items, :bought, :boolean, :default => false
```

Comment:
I'm not found of the attribute name `bought` I would suggest something like `finalized` or something. Not a blocker though